sippy-ng/src/prow_job_runs/IntervalsChart.js

```diff
@@ -468,23 +532,29 @@ export default function ProwJobRun(props) {
         Loaded {eventIntervals.length} intervals from GCS, filtered down to{' '}
         {filteredIntervals.length}.
       </p>
-      <div>
+      <div style={{ paddingBottom: '1rem' }}>
```

Comment:
nit: `paddingBottom: '1rem'` is used in three separate places, you could make a styleClass for it